### ISSUE TITLE
how to use it for windows

### ISSUE THREAD

**Issue by ChetanAmrao** [react95-io/React95-Native#57]

**Comment by arturbien** [comment]
@luizbaldi maybe you could help here 🤓

**Comment by luizbaldi** [comment]
@ChetanAmrao Sorry for taking so long here! Are you referring to using the library with [react-native-windows](https://microsoft.github.io/react-native-windows/)?

**Comment by ChetanAmrao** [comment]
> @ChetanAmrao Sorry for taking so long here! Are you referring to using the library with [react-native-windows](https://microsoft.github.io/react-native-windows/)?

Yes! I was looking for some getting started using react-native-windows for react95

**Comment by luizbaldi** [comment]
> > @ChetanAmrao Sorry for taking so long here! Are you referring to using the library with [react-native-windows](https://microsoft.github.io/react-native-windows/)?
> 
> Yes! I was looking for some getting started using react-native-windows for react95

That's quite interesting! I'd love to see how we can port that.

We'll start a discussion on the Slack channel and I'll get back to you with some info 🙇
